gnowledge/gstudio
# Added new script to logout all users along with their buddies

**Added new script to logout all users along with their buddies:**
- `doc/deployer/logout_all_users.py`
- Logging out user and making buddies free(if added any).

---

**TEST:**
- Log in from multiple browsers/sessions.
- Go to shell `python manage.py shell` and run following:
  - `execfile("../doc/deployer/logout_all_users.py")`
- All user's should be log out.
- Repeat same with buddies.
   - After running script to log out all user's with buddies, try to add same buddies in new user session, it should work.
